notification-lambda-cfn.yaml

```diff
@@ -90,4 +93,22 @@ Resources:
       FunctionName: !GetAtt Function.Arn
       Action: lambda:InvokeFunction
       Principal: events.amazonaws.com
-      SourceArn: !GetAtt EventRule.Arn
\ No newline at end of file
+      SourceArn: !GetAtt EventRule.Arn
+
+  LambdaAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions: [!Ref AlarmTopic]
+      OKActions: [!Ref AlarmTopic]
+      AlarmDescription: !Sub Triggers if there are errors persisiting on ${Stage}
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 1
+      MetricName: "Errors"
+      Namespace: "AWS/Lambda"
+      Dimensions:
+        - Name: "FunctionName"
+          Value: !Sub us-election-2020-notification-lambda-${Stage}
+      Period: 3600
+      Statistic: Sum
+      Threshold: 1
```

Comment:
We might want to consider [alerting based on error % of the lambda](https://github.com/guardian/live-app-versions/blob/main/cfn.yaml#L199-L229). The odd failure or two is probably acceptable because this lambda runs very frequently. We should really start worrying if there are consistent errors, because that's when people's notifications will start getting out of date...